.github/workflows/ci-ubuntu.yml

```diff
@@ -71,22 +65,10 @@ jobs:
 
       - name: Initialise variables
         run: |
-          if [[ '${{ github.ref }}' == 'refs/heads/master' \
-             || '${{ github.base_ref }}' == 'master' ]]; then
-            # Pick Agda version for master
-            echo "AGDA_COMMIT=tags/v2.6.1.3.20210524" >> $GITHUB_ENV;
-            echo "AGDA_HTML_DIR=html" >> $GITHUB_ENV
-          elif [[ '${{ github.ref }}' == 'refs/heads/experimental' \
-               || '${{ github.base_ref }}' == 'experimental' ]]; then
-            # Pick Agda version for experimental
-            echo "AGDA_COMMIT=9047e32a1b0cba98a299ed439a08d35bc4846f99" >> $GITHUB_ENV;
-            echo "AGDA_HTML_DIR=html/experimental" >> $GITHUB_ENV
-          fi
-
-          if [[ '${{ github.ref }}' == 'refs/heads/master' \
-             || '${{ github.ref }}' == 'refs/heads/experimental' ]]; then
-             echo "AGDA_DEPLOY=true" >> $GITHUB_ENV
-          fi
```

Comment:
So I imagine that we'd be merging this branch back into `master` at some point. While a lot of the changes to this CI file are enhancements (e.g. bumping versions), can we avoid removing functionality like this? 

In particular it's important that if we're on the experimental branch, we install a different version of Agda!

Reply:
Well on principle, I don't think we should be making any changes that we _don't_ want to merge back across. Furthermore you're making various improvements to the CI for example right? Bumping versions etc. it would nice if those were added back across. I guess we can cherry-pick them...

Reply:
The versions are already bumped in `master`, the last CI bumps are in:
- https://github.com/agda/agda-stdlib/pull/2142